### ISSUE TITLE
v0.18.1 cannot import name 'bmat' from 'scipy.sparse.sputils'

### ISSUE THREAD

**Issue by cni-md** [PennLINC/qsiprep#596]
Running v0.18.1 with singularity gave me an error:
 singularity run --cleanenv -B /pool/study:/license -B ${TMPDIR}:/tmp -B {INPUT}:/input -B {OUTPUT}:/output  /pool/container/qsiprep_v0.18.1.simg  --nthreads 80  --freesurfer-input /output/fmriprep/sourcedata/freesurfer --output-resolution 2 --hmc-model eddy  --use-syn-sdc --force_syn  --fs-license /license/license --separate_all_dwis   -w /tmp  --notrack /input /output participant
.....
  from .lobpcg import *
  File "/usr/local/miniconda/lib/python3.8/site-packages/scipy/sparse/linalg/eigen/lobpcg/lobpcg.py", line 25, in <module>
    from scipy.sparse.sputils import bmat
ImportError: cannot import name 'bmat' from 'scipy.sparse.sputils' (/usr/local/miniconda/lib/python3.8/site-packages/scipy/sparse/sputils.py)

Which is already mentioned here:
https://github.com/aleju/imgaug/issues/770

Changing the path in
/usr/local/miniconda/lib/python3.8/site-packages/scipy/sparse/linalg/eigen/lobpcg/lobpcg.py
solved this problem.

**Comment by mattcieslak** [comment]
does this problem persist if you use `--containall`? The scipy in the qsiprep container is old enough that it doesn't run into this in the CI tests

**Comment by cni-md** [comment]
@mattcieslak yes. https://github.com/PennLINC/qsiprep/issues/595#issuecomment-1624770174

**Comment by cni-md** [comment]
I checked qsiprep 0.19.0 and a different dataset. Unfortunately the same error with --containall. I guess it is a singularity problem.
